.github/workflows/build.yml

```
@@ -0,0 +1,42 @@
+name: Build and Deploy to Cloud Run
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)